<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParticleSwarmDialog</class>
 <widget class="QDialog" name="ParticleSwarmDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>288</width>
    <height>169</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Particle Swarm Options</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(255, 255, 220, 120), stop:1 rgba(255, 240, 190, 200));</string>
     </property>
     <property name="title">
      <string>Particle Swarm Options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3" columnstretch="3,0">
      <item row="5" column="0">
       <layout class="QHBoxLayout" name="m_MetricYRadiusLabelLayout_2">
        <item>
         <spacer name="m_MetricYRadiusLabelSpace_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_12">
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
          </property>
          <property name="text">
           <string>Iterations:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_42">
        <item>
         <widget class="QSpinBox" name="m_IterationsSpin">
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>10000</number>
          </property>
          <property name="singleStep">
           <number>10</number>
          </property>
          <property name="value">
           <number>1000</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="7" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_29">
        <item>
         <spacer name="horizontalSpacer_15">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_24">
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
          </property>
          <property name="text">
           <string>Type:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="6" column="0">
       <layout class="QHBoxLayout" name="m_MetricYRadiusLabelLayout_3">
        <item>
         <spacer name="m_MetricYRadiusLabelSpace_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_13">
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
          </property>
          <property name="text">
           <string># of particles:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="7" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_41">
        <item>
         <widget class="QComboBox" name="m_TypeCombo">
          <item>
           <property name="text">
            <string>Particle Swarm</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Initialization Biased</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>MultiThreaded</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="6" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_43">
        <item>
         <widget class="QSpinBox" name="m_NumberOfParticlesSpin">
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>10000</number>
          </property>
          <property name="singleStep">
           <number>10</number>
          </property>
          <property name="value">
           <number>1000</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>m_IterationsSpin</tabstop>
  <tabstop>m_NumberOfParticlesSpin</tabstop>
  <tabstop>m_TypeCombo</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ParticleSwarmDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ParticleSwarmDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>218</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
